hack/lib/init.sh

```diff
@@ -39,6 +40,7 @@ API_PACKAGES="\
 github.com/openshift/api/apps/v1,\
 github.com/openshift/api/authorization/v1,\
 github.com/openshift/api/build/v1,\
+github.com/openshift/api/cloudcredential/v1,\
```

Comment:
this package doesn't belong here. This is for the old openshift types which are served via an aggregated API.